src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -200,25 +200,34 @@
 
 (rf/reg-event-fx
  :wallet/set-token-to-send
- (fn [{:keys [db]} [{:keys [token-symbol token stack-id start-flow?]}]]
+ (fn [{:keys [db]} [{:keys [token-symbol token stack-id start-flow? owners]} entry-point]]
    ;; `token` is a map extracted from the sender, but in the wallet home page we don't know the
    ;; sender yet, so we only provide the `token-symbol`, later in
    ;; `:wallet/select-from-account` the `token` key will be set.
-   (let [{token-networks :networks}                token
-         receiver-networks                         (get-in db [:wallet :ui :send :receiver-networks])
-         token-networks-ids                        (mapv #(:chain-id %) token-networks)
-         token-not-supported-in-receiver-networks? (not-any? (set receiver-networks)
-                                                             token-networks-ids)]
-     (when (or token token-symbol)
+   (let [{:keys [networks]} token
+         receiver-networks   (get-in db [:wallet :ui :send :receiver-networks])
+         token-networks-ids  (map :chain-id networks)
+         unsupported-token?  (not-any? (set receiver-networks) token-networks-ids)
+         unique-owner        (when (= (count owners) 1)
+                               (first owners))
+         unique-owner-tokens (get-in db [:wallet :accounts unique-owner :tokens])
+         token-data          (or token
+                                 (when (and token-symbol unique-owner)
+                                   (some #(when (= (:symbol %) token-symbol) %)
+                                         unique-owner-tokens)))]
```

Comment:
Now in this event, we check if there;s only one token owner, if so, then we add the owner's token data to the send flow